include/zephyr/bluetooth/gatt.h

```diff
@@ -499,25 +530,45 @@ struct bt_gatt_ccc {
  */
 #define BT_GATT_SCC_BROADCAST                   0x0001
 
-/** Server Characteristic Configuration Attribute Value */
+/** @brief Server Characteristic Configuration Attribute Value
+ *
+ *  Used in the discovery of standard characteristic descriptor values.
+ */
 struct bt_gatt_scc {
-	/** Server Characteristic Configuration flags */
+	/** Server Characteristic Configuration flags, a bitmap of ``BT_GATT_SCC_*`` macros. */
 	uint16_t flags;
 };
 
-/** @brief GATT Characteristic Presentation Format Attribute Value. */
+/** @brief GATT Characteristic Presentation Format Attribute Value.
+ *
+ *  Used in the discovery of standard characteristic descriptor values. Can be
+ *  used with the @ref BT_GATT_CPF macro to declare the CPF descriptor.
+ */
 struct bt_gatt_cpf {
-	/** Format of the value of the characteristic */
+	/** Format of the value of the characteristic. The format types can be
+	 *  found in section 2.4.1 of the Bluetooth SIG Assigned Numbers document.
+	 */
 	uint8_t format;
 	/** Exponent field to determine how the value of this characteristic is
-	 * further formatted
+	 *  further formatted. Only used on integer format types.
+	 *  actual value = Characteristic Value x 10^Exponent
 	 */
 	int8_t exponent;
-	/** Unit of the characteristic */
+	/** UUID of the unit of the characteristic. The units can be found in
+	 *  section 3.5 of the Bluetooth SIG Assigned Numbers document.
+	 */
 	uint16_t unit;
-	/** Name space of the description */
+	/** Name space of the description, used to identify the organization
+	 *  that is responsible for defining the enumerations for the
+	 *  description field. See section 2.4.2 of the Bluetooth SIG Assigned
+	 *  Numbers document.
+	 */
 	uint8_t name_space;
-	/** Description of the characteristic as defined in a higher layer profile */
+	/** Description of the characteristic as defined in a higher layer
+	 *  profile. An enumerated value defined by the organization identified
+	 *  by the name_space field. See section 2.4.2.1 of the Bluetooth SIG
+	 *  Assigned Numbers document.
+	 */
```

Comment:
The same applies here. The first sentence (brief) should ideally be very brief and any additional information should be it's own paragraph